### ISSUE TITLE
Error when connecting to a GCE (Google ComputeEngine) node via RemoteSetUnion

### ISSUE THREAD

**Issue by achimnol** [spoqa/geofront#32]
Using RemoteSetUnion, I could list all instances across AWS and GCE, which are federated via IPsec VPN, using specific filter conditions. 😄 (But this worked only after rolling back `GroupMetadataPermissionPolicy` to `DefaultPermissionPolicy` -- but I was going to deprecate it anyway in my setup. This may be related to #25 where I mentioned that libcloud does not provide an API to "get" node metadata for GCE.)
However, connecting to GCE instances via HTTP WebSocket proxy (`-t` option in my fork branch) does not work with an internal server error. 😞 

Unfortunately, there is no detailed traceback right now due to missing implementation of #31...
The root cause might be some incompatibility inside libcloud. I will try to get more information later!

**Comment by achimnol** [comment]
I just found that GCE controls the intial user name using the "comment" area of the SSH keyphrase (`username` part in `ssh-rsa AAAAB3NzaC...yw== username`) instead of cloud-init user-data.
And it works!

I think this incompatibility is out of scope of Geofront, but still I'd suggest to add some hints & guides in the documentation.
